liquibase-service/src/main/resources/db/rdb/037-generate_data_for_data_exchange_config.sql

```diff
@@ -0,0 +1,35 @@
+INSERT INTO [RDB].[dbo].[data_exchange_config] (table_name, source_db, query, query_with_limit) VALUES
```

Comment:
@alog-enquizit Will Liquibase try to insert the rows again if there is no check?

Reply:
@upasanapattnaik-eq , It will be run only once in the lifetime of the database.  This is similar to the Table Creations.

Any new Inserts, updates or delete, should be appended to a new liquibase script file

Reply:
Got it. So any new columns will need the alter add columns?

Reply:
so we dont need any check for these insertion?

Reply:
then these are good to go as they are, there will be several more

Reply:
Yes this is good as they are!